app/controllers/patients_controller.rb

```diff
@@ -716,28 +716,28 @@ def allowed_params
   def group_member_subset
     %i[
       address_line_1
-      foreign_address_line_1
+      address_line_2
```

Comment:
I see that the `foreign_address_line_1` field was out of order, but would prefer to keep the order of these fields consistent (throughout the back end at least):

https://github.com/SaraAlert/SaraAlert/blob/master/test/system/roles/enroller/enrollment/steps.rb#L29

https://github.com/SaraAlert/SaraAlert/blob/master/app/lib/import_export.rb#L59

Reply:
reverted except for the case you mentioned: https://github.com/SaraAlert/SaraAlert/pull/737/commits/c7e3e553d0885f6e063ade0c3213ed6c31971a69